syntax = "proto2";

message AuthenticationResponse{
    enum Status {OK=0; FAIL=1;}
    required Status status = 1;	
}
